### ISSUE TITLE
Compilation failure on Windows 10

### ISSUE THREAD

**Issue by mchamberland** [clrp-code/egs_brachy#3]
See report [here](https://plus.google.com/118233686303173529599/posts/8prcPEX6hRL?fscid=z13psf2ibnawtzt1i22dw1r5wwqivpubs04.1515431617182625). Unfortunately, I have no access to Windows 10 to investigate this.

Anecdotally, the EGSnrc repo had a couple of issues related to the mingw compiler and what I vaguely remember is that its trouble-prone. : /

**Comment by mchamberland** [comment]
The original failure was due to the SendMessage type conflicting with an already defined type on Windows. Workaround is to replace every instance of SendMessage with EBSendMessage (or any meaningful name!).

Spots to replace:

```
./egs_brachy.cpp:1480:    SendMessage send_messages[] = {
./egs_brachy.cpp:1481:        SendMessage(is_before_transport, PARTICLE_TAKING_STEP),
./egs_brachy.cpp:1482:        SendMessage(is_after_transport, PARTICLE_TOOK_STEP),
./egs_brachy.cpp:1483:        SendMessage(escaping_source, PARTICLE_ESCAPING_SOURCE),
./egs_brachy.cpp:1484:        SendMessage(escaped_source, PARTICLE_ESCAPED_SOURCE),
./egs_brachy.cpp:1485:        SendMessage(escaping_geom, PARTICLE_ESCAPING_GEOM),
./egs_brachy.cpp:1486:        SendMessage(escaped_geom, PARTICLE_ESCAPED_GEOM)
./pubsub.h:69:typedef pair<bool, EB_Message> SendMessage;

```

**Comment by mchamberland** [comment]
This specific issue is now fixed with ea625617048e26ffd62be494112fa33a3e494865.
